dedupe/core.py

```diff
@@ -121,6 +121,7 @@ def mergeScores(score_queue, result_queue, stop_signals) :
 
     seen_signals = 0
     start = 0
+    end = 0
```

Comment:
This is actually possible.
`score_queue` has only `None` because of a high threshold, and `score_chunk = score_queue.get()` gives out only a `None`.

In this case, the line that defines `end` (https://github.com/dedupeio/dedupe/blob/master/dedupe/core.py#L139) is never reached.